awsenergylabelercli/awsenergylabelercli.py

```diff
@@ -82,22 +89,24 @@ def get_arguments():
                         action='store',
                         dest='logger_config',
                         help='The location of the logging config json file',
-                        default='')
+                        default=os.environ.get('LOG_CONFIG', ''))
     parser.add_argument('--log-level',
                         '-L',
                         help='Provide the log level. Defaults to info.',
                         dest='log_level',
                         action='store',
-                        default='info',
+                        default=os.environ.get('LOG_LEVEL', 'info'),
```

Comment:
LOG_LEVEL is a bit too generic to assume it is targeting this specific software. If we want to support env vars for this option I would suggest something a bit more opinionated to make sure we are not picking up irrelevant settings intended for other software.